libr/core/project.c

```diff
@@ -438,25 +464,23 @@ R_API bool r_core_project_open(RCore *core, const char *prjfile, bool thready) {
 		r_io_desc_init (core->io);
 		if (filepath[0]) {
 			/* Old-style project without embedded on commands to open all files.  */
-			if (!r_core_file_open (core, filepath, 0, 0)) {
+			if (!r_core_file_open (core, filepath, 0, UT64_MAX)) {
 				eprintf ("Cannot open file '%s'\n", filepath);
 				ret = false;
 				goto beach;
 			}
 		}
 	}
-
+	mapaddr = getProjectLaddr(core, prj);
+	if (mapaddr) {
+		r_config_set_i (core->config, "bin.laddr", mapaddr);
+	}
 	if (filepath[0] && close_current_session && r_config_get_i (core->config, "file.info")) {
 		mapaddr = r_config_get_i (core->config, "file.offset");
 		(void)r_core_bin_load (core, filepath, mapaddr? mapaddr: UT64_MAX);
 	}
-	if (thready) {
```

Comment:
why kill the thready project loading support?

Reply:
already killed at line 404